src/main/java/Deadline.java

```diff
@@ -0,0 +1,33 @@
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+
+public class Deadline extends Task {
+    protected String by;
+    protected LocalDate dueDate;
+    protected LocalTime timing;
+
+    /**
+     * Constructor for Deadline Class.
+     *
+     * @param description Deadline description.
+     * @param by          Due date for the deadline.
+     */
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+        String[] info = by.split(" ");
```

Comment:
you may want to separate out the parsing to a different method, thereby improving the SLAP